sources/platform/actors/running/actor_standby.md

```diff
@@ -0,0 +1,62 @@
+it well. Please, head to Actor README to learn more about the capabilities of your chosen Actor.
+
+#### Is there any scaling to accommodate the incoming requests?
+When you use the Actor in the Standby mode, the system automatically scales the Actor to accommodate the incoming requests. Under the hood,
+the system starts new Actor runs, which you will see in the Actor runs tab, with origin set to Standby.
+
+#### How do I customize standby configuration?
+The Standby configuration currently consists of the following properties
+- **Max requests per run** - The maximum number of concurrent HTTP requests a single standby Actor run can accept. If this limit is exceeded, the system starts a new Actor run to handle the request, which will take a few seconds.
+- **Desired requests per run** - A desired number of concurrent HTTP requests a single standby Actor run can handle. If this limit is exceeded, the system preemptively starts a new Actor run to handle additional requests.
+- **Memory (MB)** - The amount of memory (RAM) allocated for the Actor standby run, in megabytes. With more memory, the Actor can typically handle more requests in parallel, but the number of compute units consumed and the cost also grow.
+- **Idle timeout (seconds)** - If a standby Actor run doesn’t receive any HTTP requests in this amount of time, the system will finish the run. When a new request comes, the system might need to start a new standby Actor run to handle it, which will take a few seconds. Higher idle timeout increases the responsiveness but increases the costs as the Actor needs to keep running longer.
+
+You can see these in the Standby tab of the Actor detail page. However, note that these properties are not configurable at the Actor level. If you wish to
+use the Actor-level hostname, this will always use the default configuration. In order to override this configuration, just create a new Task from the Actor.
+You can then head to the Standby tab of the created Task and modify the configuration as needed. Note that the task has specific hostname, so make
+sure to use that in your application if you wish to use the custom configuration.
+
+#### Are the Standby runs billed differently?
+No, the standby runs are billed in the same fashion as the normal runs.
+
+#### Are the Standby runs shared among users?
+No, even if you use the Actor-level hostname with the default configuration, the background Actor runs for your requests are not shared with other users.
+
+#### How can I enable standby for my Actor?
+You can head to Settings tab of your Actor, enabled Standby mode and set the default configuration.
+![Standby for creators](./images/actor_standby/standby-creators.png)
+
+Please make sure that you properly describe your Actors, its endpoints, and the schema for their
+inputs and ouputs in your README.
+
+#### Can I monetize my Actor in the Standby mode?
+No, the Standby mode is currently in Beta and the monetization is not supported.
```

Comment:
[To consider]: I am thinking that some readme example would be nice, with [#actor-standby header](https://github.com/apify/apify-core/pull/16488#discussion_r1661270287)